src/Hub.h

```diff
@@ -137,25 +137,32 @@ struct Hub {
 
     sc2::UNIT_TYPEID GetCurrentWorkerType() const;
 
+    sc2::UNIT_TYPEID GetCurrentTownHallType() const;
+
     bool AssignRefineryConstruction(Order* order_, const sc2::Unit* geyser_);
 
-    bool AssignBuildTask(Order* order_, const sc2::Point2D& point_);
+    sc2::Tag AssignBuildTask(Order* order_, const sc2::Point2D& point_);
 
     void AssignVespeneHarvester(const sc2::Unit& refinery_);
 
+    void DistrubuteMineralWorker(const sc2::Unit* unit_);
+
     bool AssignLarva(Order* order_);
 
     const Cache<GameObject>&  GetLarvas() const;
 
     const Expansions& GetExpansions() const;
 
-    const sc2::Point3D* GetNextExpansion();
+    Expansion* GetNextExpansion();
+    
+    void UpdateExpansionOwner(const sc2::Unit& unit_);
```

Comment:
Seems that this method should be at least private as it is not used outside of this class.

Reply:
Done.